### DISCUSSION TITLE
How to include radiation heat loss in PeleLMeX

### DISCUSSION THREAD

**Discussion by Alex-jian522** [AMReX-Combustion/PeleLMeX#425] [General]
Hello, everyone!
     I would like to include radiation heat loss in PeleLMeX, but I can not find any information to achieve it in Pele. Does anyone know how can I do this? Thanks very much for your reply.

**Comment by wjge** [comment]
You can find one example at: https://github.com/AMReX-Combustion/PeleLMeX/tree/development/Exec/RegTests/SootRadTest

**Reply by Alex-jian522** [reply]
Thank you very much for your help. I checked the link and found that the document is still very simple. And he focuses on the radiation of soot? I am more interested in the radiation between gases (without soot). Anyway, I would like to express my gratitude again.
